bundles/org.openhab.binding.miio/src/main/resources/.gitignore

```diff
@@ -0,0 +1 @@
+/logback-test.xml
```

Comment:
This is unexpected from previous comments. Can you revert this file.

Reply:
I'll address this once I'm back from holidays in 2 weeks

Reply:
Are you able to address this ?

Reply:
totally forgot this indeed... will fix

Reply:
@lsiepel thanks for the reminder. This should be fixed now